database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -12,12 +12,15 @@ class CreateUsersTable extends Migration {
 	 */
 	public function up()
 	{
-		Schema::create('users', function(Blueprint $table)
-		{
+		Schema::create('users', function (Blueprint $table) {
 			$table->increments('id');
+
+			$table->string('github_id')->unique();
 			$table->string('name');
-			$table->string('email')->unique();
-			$table->string('password', 60);
+			$table->string('email');
+			$table->string('avatar');
+			$table->string('token');
```

Comment:
Since we have live data already, you'll have to add this modification as a new migration; otherwise it won't be run on the live site.